consts.php

```diff
@@ -107,12 +107,131 @@
     ],
 ];
 
+// use hardcoded.php to bulk update update this after creating a .cow.pat.json
+// for multiple versions, use an array e.g. silverstripe-mymodule => ['2.3', '2.4']
 const INSTALLER_TO_REPO_MINOR_VERSIONS = [
     '4.10' => [
```

Comment:
Since we're not doing this by hand anymore, is it worth revisiting the structure of this const as per [this comment](https://github.com/silverstripe/gha-generate-matrix/pull/12#discussion_r918390614)?
I'm not too fussed either way - happy to keep as is if you are given that it does the job.

Reply:
This structure still makes sense to me, likely it'll get manual updates later on e.g. silverstripe-html5 has 2 versions for 4.10